source/Halibut.Tests/PublicSurfaceAreaFixture.ThePublicSurfaceAreaShouldNotRegress.NETCore.approved.cs

```diff
@@ -167,7 +168,7 @@ public interface ILogFactory
     }
     public class InMemoryConnectionLog : Halibut.Diagnostics.ILog
     {
-        public InMemoryConnectionLog(string endpoint) { }
+        public InMemoryConnectionLog(string endpoint, Halibut.Diagnostics.LogEventStorage logEventStorage) { }
```

Comment:
Breaking API change, we should bump the version number via commit message

Reply:
Ahhh, I didn't think this would be a breaking change, but you're right. TBH if we're gonna break the API I'd change the logging api so we don't have to hold onto the format objects of the logs, which is what's causing this problem in the first place.